config/initializers/session_store.rb

```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Helpcue::Application.config.session_store :cookie_store, key: '_helpcue_session'
+Rails.application.config.session_store :cookie_store, key: '_helpcue_session'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.